src/Umbraco.Cms.Integrations.Analytics.Cookiebot/umbraco-marketplace.json

```diff
@@ -0,0 +1,51 @@
+{
+  "AuthorDetails": {
+    "Description": "Umbraco HQ",
+    "Url": "https://umbraco.com/",
+    "ImageUrl": "https://avatars.githubusercontent.com/u/1419552?s=200&v=4"
+  },
+  "Category": "Analytics & Insights",
+  "LicenseType": "Free",
+  "PackageType": "Integration",
+  "PackagesByAuthor": [
+    "Umbraco.Cms.Integrations.Commerce.CommerceTools",
+    "Umbraco.Cms.Integrations.Commerce.Shopify",
+    "Umbraco.Cms.Integrations.SEO.GoogleSearchConsole.URLInspectionTool",
+    "Umbraco.Cms.Integrations.Crm.Hubspot",
+    "Umbraco.Cms.Integrations.Crm.Dynamics",
+    "Umbraco.Cms.Integrations.Crm.ActiveCampaign",
+    "Umbraco.Cms.Integrations.Automation.Zapier"
+  ],
+  "RelatedPackages": [
```

Comment:
I think we can remove these, as they aren't really related, other than being created by Umbraco HQ, and `PackagesByAuthor` will cover that.